transpire/internal/config.py

```diff
@@ -195,3 +195,18 @@ def from_cwd(cls, cwd: Path = Path.cwd()) -> "ClusterConfig":
                 "cluster.toml not found up to current git or fs boundary"
             )
         return cls.from_cwd(cwd.parent)
+
+def get_config(module_name: str | None = None, cwd: Path = Path.cwd()) -> ModuleType:
+    config = ".transpire.py"
+    name = "_transpire"
+    if module_name:
+        config = "cluster.toml"
+        name = module_name
+    config_path = cwd / config
+    if config_path.exists():
+        return load_py_module_from_file(name, config_path, name)
```

Comment:
I think this is a logic bug. If the name is specified, this will try to load `cluster.toml` as a Python module, which won't work. Your original logic with code duplication is probably cleaner.